pep-0467.txt

```diff
@@ -189,11 +174,28 @@ that ``bytes(x)`` (where ``x`` is an integer) should behave like the
 class methods avoids that ambiguity.
 
 
+Omitting the originally proposed builtin function
+-------------------------------------------------
+
+When submitted to the Steering Council, this PEP proposed the introduction of
+a ``bchr`` builtin (with the same behaviour as ``bytes.fromint``), recreating
+the ``ord``/``chr``/``unichr`` trio from Python 2 under a different naming
+scheme (``ord``/``bchr``/``chr``).
+
+The SC indicated they didn't think this functionality was needed often enough
+to justify offering two ways of doing the same thing, especially when one of
+those ways was a new builtin function. That part of the proposal was therefore
+dropped as being redundant with the ``bytes.fromint`` alternate constructor.
+
+Developers that use this method frequently will instead have the option to
+define their own ``bchr = bytes.fromint`` aliases.
+
 
-Open Issue: memoryview
-======================
+Scope limitation: memoryview
+----------------------------
 
-Updating ``memoryview`` with these new methods is outside the scope of this PEP.
+Updating ``memoryview`` with the new item retrieval methods is outside the scope
+of this PEP.
```

Comment:
This section looked weird to me when I was adding the new design discussion entry, so I tweaked it a bit:
- it's a deliberate scope limitation, not an open issue
- only the item retrieval methods (`getbyte()`, `iterbytes()`) potentially make sense for memory views, the alternate constructors would never be relevant